packages/office-ui-fabric-react/src/components/__snapshots__/Pivot.TabsLarge.Example.tsx.shot

```diff
@@ -597,7 +597,7 @@ exports[`Component Examples renders Pivot.TabsLarge.Example.tsx correctly 1`] =
                 top: 0px;
               }
               &:hover {
-                background-color: #f3f2f1;
+                background-color: #ffffff;
```

Comment:
Hmmm... why on hover we are having a white background? @marygans

Reply:
Putting in a new PR after this to update to new semantic slot for background-color to `buttonBackground`